content/src/Blip.Api.Template/Interfaces/IContentProviderAPI.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Blip.Api.Template.Models.NLPContentProviderModels;
+using RestEase;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Blip.Api.Template.Interfaces
+{
+    public interface IContentProviderAPI : IDisposable
```

Comment:
As this template aims for a general purpose template, for people working @ Take and people not working there (hence no access to ContentProvider), this should not be here but should be, instead, a plug-and-play NuGet package for people who work there to use